apps/avifenc.c

```diff
@@ -805,6 +792,63 @@ int main(int argc, char * argv[])
         ++argIndex;
     }
 
+    // Check lossy/lossless parameters and set to default if needed.
+    if (lossless) {
+        // Pixel format.
+        if (input.requestedFormat != AVIF_PIXEL_FORMAT_NONE && input.requestedFormat != AVIF_PIXEL_FORMAT_YUV444) {
+            fprintf(stderr,
+                    "When set, the pixel format can only be 444 in lossless "
+                    "mode.\n");
+            returnCode = 1;
+        }
+        input.requestedFormat = AVIF_PIXEL_FORMAT_YUV444; // don't subsample when using
+                                                          // AVIF_MATRIX_COEFFICIENTS_IDENTITY
+        // Quantizers.
+        if (minQuantizer != -1 || maxQuantizer != -1 || minQuantizerAlpha != -1 || maxQuantizerAlpha != -1) {
+            fprintf(stderr, "Quantizers cannot be set in lossless mode.\n");
+            returnCode = 1;
+        }
+        minQuantizer = maxQuantizer = minQuantizerAlpha = maxQuantizerAlpha = AVIF_QUANTIZER_LOSSLESS;
+        // Codec.
+        if (codecChoice != AVIF_CODEC_CHOICE_AUTO && codecChoice != AVIF_CODEC_CHOICE_AOM) {
+            fprintf(stderr, "Codec can only be AOM in lossless mode.\n");
+            returnCode = 1;
+        }
+        codecChoice = AVIF_CODEC_CHOICE_AOM; // rav1e doesn't support lossless
+                                             // transform yet:
+                                             // https://github.com/xiph/rav1e/issues/151
+                                             // SVT-AV1 doesn't support lossless
+                                             // encoding yet:
+                                             // https://gitlab.com/AOMediaCodec/SVT-AV1/-/issues/1636
```

Comment:
Nit: Similarly, I suggest moving the comment before the assignment statement:

```
        // rav1e doesn't support lossless transform yet:
        // https://github.com/xiph/rav1e/issues/151
        // SVT-AV1 doesn't support lossless encoding yet:
        // https://gitlab.com/AOMediaCodec/SVT-AV1/-/issues/1636
        codecChoice = AVIF_CODEC_CHOICE_AOM;
```

Reply:
Done